src/Service.GraphQLBuilder/GraphQLNaming.cs

```diff
@@ -30,6 +30,11 @@ public static class GraphQLNaming
 
         public const string LINKING_OBJECT_PREFIX = "linkingObject";
 
+        /// <summary>
+        /// For any CosmosDB Query, the default alias for the container is 'c'
```

Comment:
Consider making this a public constant in CosmosQueryStructure. Then it can be used instead of this and can also be used instead of the private var in CosmosQueryBuilder.